app/views_classes.py

```diff
@@ -17,18 +17,6 @@ def __init__(self, index, play, library, music, playlist, title, artist,
     def __getitem__(self, index, play, library, music, playlist, title, artist,
                     album, release_date, youtube_id, artist_id, album_id):
         return self.index
```

Comment:
I'm not sure what the intent is of these multiple return statements. `return` immediately exits the current scope, so the multiple `return`s after the first one can never be reached.